Makefile

```diff
@@ -69,4 +69,4 @@ lint:
 locales:
 	$(VIRTUAL_ENV)/bin/python manage.py makemessages -d djangojs
 	$(VIRTUAL_ENV)/bin/python manage.py makemessages -d django
-	$(VIRTUAL_ENV)/bin/python manage.py compilemessages
+	$(VIRTUAL_ENV)/bin/python manage.py compilemessages
```

Comment:
There is no need to modify this file. Also `.editorconfig` says that there should always be a newline.